### PR TITLE
(AzureCXP) fix the permissions description

### LINKED ISSUE #26542: Incorrect description for the permissions parameter.
[Enter feedback here]
The permissions parameter description incorrectly repeats the description of the expiresOn parameter. 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: af699d50-6ad8-f434-c93b-86e350520d7c
* Version Independent ID: 030b007d-8a0f-70f3-0d65-144e06eee25a
* Content: [BlobSasBuilder Constructor (Azure.Storage.Sas) - Azure for .NET Developers](https://docs.microsoft.com/en-us/dotnet/api/azure.storage.sas.blobsasbuilder.-ctor?view=azure-dotnet#azure-storage-sas-blobsasbuilder-ctor(azure-storage-sas-blobcontainersaspermissions-system-datetimeoffset))
* Content Source: [xml/Azure.Storage.Sas/BlobSasBuilder.xml](https://github.com/Azure/azure-docs-sdk-dotnet/blob/master/xml/Azure.Storage.Sas/BlobSasBuilder.xml)
* Service: **azure**
* GitHub Login: @rloutlaw
* Microsoft Alias: **routlaw**

### PR DESCRIPTION
**Pull request by navba-MSFT**
(AzureCXP) fix the permission's parameter description

# All SDK Contribution checklist:

This checklist is used to make sure that common guidelines for a pull request are followed.
- [x] **Please open PR in `Draft` mode if it is:**
	- Work in progress or not intended to be merged.
	- Encountering multiple pipeline failures and working on fixes.
- [ ] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/breaking-change-rules.md).**

### [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#testing-guidelines)
- [ ] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#sdk-generation-guidelines)
- [ ] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code. (Track 2 only)
- [ ] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK. Please double check nuget.org current release version.

## Additional management plane SDK specific contribution checklist: 
Note: Only applies to `Microsoft.Azure.Management.[RP]` or `Azure.ResourceManager.[RP]`
 
- [ ] Include updated [management metadata](https://github.com/Azure/azure-sdk-for-net/tree/main/eng/mgmt/mgmtmetadata).
- [ ] Update AzureRP.props to add/remove version info to maintain up to date API versions.

### Management plane SDK Troubleshooting
- If this is very first SDK for a services and you are adding new service folders directly under /SDK, please add `new service` label and/or contact assigned reviewer.
- If the check fails at the `Verify Code Generation` step, please ensure:
	- Do not modify any code in generated folders.
	- Do not selectively include/remove generated files in the PR.
	- Do use `generate.ps1/cmd` to generate this PR instead of calling `autorest` directly.
	Please pay attention to the @microsoft.csharp version output after running `generate.ps1`. If it is lower than current released version (2.3.82), please run it again as it should pull down the latest version.
	
	**Note: We have recently updated the PSH module called by `generate.ps1` to emit additional data. This would help reduce/eliminate the Code Verification check error. Please run following command**:

	    `dotnet msbuild eng/mgmt.proj /t:Util /p:UtilityName=InstallPsModules`

### Old outstanding PR cleanup
 Please note:
	If PRs (including draft) has been out for more than 60 days and there are no responses from our query or followups, they will be closed to maintain a concise list for our reviewers.

Fixes #26542

### CODE REVIEW

File: sdk/storage/Azure.Storage.Blobs/src/Sas/BlobSasBuilder.cs [hunk 1]
```diff
@@ -201,8 +201,9 @@ public BlobSasBuilder()
         /// class to create a Blob Service Sas.
         /// </summary>
         /// <param name="permissions">
-        /// The time at which the shared access signature becomes invalid.
-        /// This field must be omitted if it has been specified in an
+        /// The permissions associated with the shared access signature.
```
Comment (seanmcc-msft) [inline sdk/storage/Azure.Storage.Blobs/src/Sas/BlobSasBuilder.cs:204]: We need to remove the trailing whitespace so CI can pass.

**Review by navba-MSFT — COMMENTED** [review]
Fixed the permissions description and removed the trailing whitespaces

File: sdk/storage/Azure.Storage.Blobs/src/Sas/BlobSasBuilder.cs (hunk 1 above)
Comment (navba-MSFT) [reply sdk/storage/Azure.Storage.Blobs/src/Sas/BlobSasBuilder.cs]: Removed the trailing whitespaces. Please review [this](https://github.com/Azure/azure-sdk-for-net/pull/26570/commits/66865fbe352a467f554f60016f5769c8e9973906) commit once you get a chance.

**Review by seanmcc-msft — APPROVED** [review]

**Comment by navba-MSFT** [comment]
@seanmcc-msft Thanks for approving the PR. Could you merge this with the main branch ?

**Pull request merged**